--- /home/matheus/Documents/UFPE/merge-and-code-review/project/mergiraf-semi/examples/swift/Math/expected.swift	2025-11-27 23:07:07.185093249 -0300
+++ /home/matheus/Documents/UFPE/merge-and-code-review/project/mergiraf-semi/examples/swift/Math/report/merged_mergiraf.swift	2025-11-28 00:15:16.613350908 -0300
@@ -1,11 +1 @@
-public class Math {
-    public func sum(a: Int, b: Int) -> Int {
-        return a + b
-    }
-    public func subtract(a: Int, b: Int) -> Int {
-        return a - b
-    }
-    public func multiply(a: Int, b: Int) -> Int {
-        return a * b
-    }
-}
\ No newline at end of file
+// cargo failed with exit 134. See merged_mergiraf.swift.stderr
