--- /home/matheus/Documents/UFPE/merge-and-code-review/project/mergiraf-semi/examples/swift/property_observers/expected.swift	2025-11-27 23:13:54.220870971 -0300
+++ /home/matheus/Documents/UFPE/merge-and-code-review/project/mergiraf-semi/examples/swift/property_observers/report/merged_diff3.swift	2025-11-28 00:15:20.157385001 -0300
@@ -1,12 +1 @@
-struct Person {
-    var id: String
-    var name: String
-    var nicknames: [String] = ["Ada"] {
-        willSet {
-            print("Will set nicknames to \(newValue)")
-        }
-        didSet {
-            print("Did set nicknames from \(oldValue)")
-        }
-    }
-}
+// diff3 (git merge-file) failed with exit 1. See merged_diff3.stderr
